src/workers.rs

```diff
@@ -16,7 +16,7 @@ pub async fn start_workers() {
     queue.connect(NoTls).await.unwrap();
 
     let params = SleepParams {
-        sleep_period: 1,
+        sleep_period: 0,
```

Comment:
are you sure it's a good approach?

your bot will be using a lot of cpu resources even when there are no messages to process

Reply:
Just testing , i will put back in the original value